lib/src/utf.dart

```diff
@@ -0,0 +1,252 @@
+// Large portions of this code where taken from https://github.com/dart-lang/utf
```

Comment:
Hmmm ... we are in a strange world if we're now copying package code into clients rather than depending on it. UTF-8 decoding is non-trivial and it's possible we'd have bugs (including security issues) by having a copy.

Can you explain more about the rationale here?

(Would "dart:convert" work here as an alternative?)

Reply:
I have zero clue. `utf` is doing some magic to pull out the Unicode code points (I think?), which is not something you can do with `dart:convert` AFAIK – @lrhn ?

Reply:
Moreover, I'd like to just discontinue `pkg:utf`. It has no support, etc.

Reply:
gotcha. SGTM then